assignments/lambda-classes.js

```diff
@@ -1 +1,110 @@
 // CODE here for your Lambda Classes
+
+class Person{
+    constructor(attrs){
+        this.name = attrs.name
+        this.age = attrs.age
+        this.location = attrs.location
+    }
+    speak(){
+        return `Hello my name is ${this.name}, I am from ${this.location}`
+    }
+}
+
+class Teacher extends Person{
+    constructor(attrs){
+        super(attrs)
+        this.specialty = attrs.specialty
+        this.favLanguage = attrs.favLanguage
+        this.catchPhrase = attrs.catchPhrase
+    }
+    demo(string){
+        return `Today we are learning about ${string}`
+    }
+    grade(student, subject){
+        return `${student} recieves a score perfect on ${subject}`
+    }
+}
+
+class Student extends Person{
+    constructor(attrs){
+        super(attrs)
+        this.previousBackground = attrs.previousBackground
+        this.className = attrs.className
+        this.favSubjects = attrs.favSubjects
+        }
+
+    listsSubjects(){
```

Comment:
Got it! This works if you string interpolate the element parameter:

listsSubjects(){
    this.favSubjects.forEach(element => {
        console.log(`${element} is ${this.name}'s fave subject.`)
    });
}

Reply:
It won't show the backticks for some reason.